knimin/lib/data_access.py

```diff
@@ -703,7 +708,8 @@ def format_survey_data(self, md):
             md[1][barcode]['SUBSET_IBD'] = \
                 md[1][barcode]['IBD'] == 'I do not have this condition'
             md[1][barcode]['SUBSET_ANTIBIOTIC_HISTORY'] = \
-            md[1][barcode]['ANTIBIOTIC_HISTORY'] == 'I have not taken antibiotics in the past year.'
+                md[1][barcode]['ANTIBIOTIC_HISTORY'] == \
```

Comment:
parenthesis instead of a `/` line break here might make it clearer what's going on